backend/elasticsearch/src/main/java/org/hibernate/search/backend/elasticsearch/types/sort/impl/StandardFieldSortBuilderFactory.java

```diff
@@ -54,4 +60,15 @@ public boolean isDslCompatibleWith(ElasticsearchFieldSortBuilderFactory obj) {
 
 		return converter.isDslCompatibleWith( other.converter );
```

Comment:
Same here for the sort  builder factories, we should probably check for the "sortable" parameter to be compatible.

By the way, "comparing" here is a bit tricky, as `DEFAULT` is, theroretically, compatible with `NO`... So before we compare the values with each other, I think we should convert "default" to whatever the default is... Maybe we should use a boolean at this point? If so, I think it should be done in the `*TypeContext`, where we already have a switch on the enum: just add a `boolean sortableDefaulted` variable and populate it in the switch.

Reply:
Right, fixed.